content/vendor/guides/configuring-application-snapshots.md

```diff
@@ -0,0 +1,660 @@
+---
+date: "2020-09-06T00:00:00Z"
+lastmod: "2020-09-06T00:00:00Z"
+title: "Configure Application Snapshots"
+weight: "1005"
+---
+
+KOTS provides [snapshot capabilities](https://kots.io/vendor/snapshots/overview/) by leveraging the [Velero](https://velero.io/) open source project. This is an advanced topic 
+
+## Objective 
+
+The purpose of this guide is to walk you through the nescessary steps to enable and configure snaphots in KOTS. 
+In this guide, we'll use a Postgres `StatefulSet` as this is a common use case.
+
+## Prerequisites & Assumptions
+
+This power-user's guide assumes you have completed the [standard quickstart](https://kots.io/vendor/guides/quickstart/) or the [CLI quickstart](https://kots.io/vendor/guides/cli-quickstart/) guides as this guide is a continuation of those guides.
+As with the previous guides, we will also need a VM to install the application with the following minimim requirements:
+
+* Ubuntu 18.04
+* At least 8 GB of RAM
+* 4 CPU cores
+* At least 100GB of disk space
+
+## Overview
+
+The guide is divided into the following steps:
+
+  [1. Set Up Testing Environment](#1-set-up-testing-environment) 
+
+  [2. Enable Snashots in KOTS](#2-enable-snashots-in-kots) 
+  
+  [3. Configure the Deployed Application](#3-configure-the-deployed-application) 
+  
+  [4. Take the First Snaphots](#4-take-the-first-snaphots) 
+  
+  [5. Add Another Record to the Table](#5-add-another-record-to-the-table) 
+  
+  [6. Restore the First Snapshot](#6-restore-the-first-snapshot)
+  
+  [7. Verify Table Contents](#7-verify-table-contents)
+  
+  [Optional/Advanced Use Case: Backup Hooks](#optionaladvanced-use-case-backup-hooks)
+  
+  [Appendix A - Manifest Files](#appendix-a---manifest-files)
+
+## 1. Set Up Testing Environment
+
+For this guide, we are simply deploying a Postgres `StatefulSet` and then will use `psql` to connect, to create a table and add records.
```

Comment:
small style thing, I think the `to` should be uniform across items in the list. this could either be 

> use `psql` to connect, to create a table, and to add records

or 

> use `psql` to connect, create a table, and add records


```suggestion
For this guide, we are simply deploying a Postgres `StatefulSet` and then will use `psql` to connect, create a table and add records.
```